### ISSUE TITLE
Doubled quotes in ValueRef::fmt

### ISSUE THREAD

**Issue by vorner** [3Hren/msgpack-rust#113]
When string values (the `ValueRef` type) are printed with the `Display` trait, the double quotes around strings are doubled. It can be demonstrated by this program:

```rust
extern crate rmp_serde;
extern crate rmpv;

use std::collections::HashMap;

use rmp_serde::to_vec;
use rmpv::decode::read_value_ref;

fn main() {
    let mut data = HashMap::new();
    data.insert("field".to_owned(), "value".to_owned());
    let vec = to_vec(&data).unwrap();
    let val = read_value_ref(&mut vec.as_slice()).unwrap();
    println!("{}", val);
}
```

It prints:

```
{""field"": ""value""}
```

I guess the intention was (or at least I expected the output to be):

```
{"field": "value"}
```

The ordinary `Value` prints correctly.

**Comment by 3Hren** [comment]
Fixed in https://github.com/3Hren/msgpack-rust/commit/7ef9242c9d1a50871b8c47b433da200281f65fe7, will publish right now.

**Comment by 3Hren** [comment]
Thanks for the report!

Landed in rmpv v0.3.2: https://crates.io/crates/rmpv/0.3.2

**Comment by vorner** [comment]
Cool, that was fast :-).
